assets/js/helpers.js

```diff
@@ -116,7 +116,9 @@ function updateLists() {
 }
 
 function logout() {
+  var theme = localStorage.theme;
   localStorage.clear();
```

Comment:
Could just remove jwt instead of clearing everything?

Reply:
Agreed